Experimental/src/org/sleuthkit/autopsy/experimental/volatilityDSP/VolatilityProcessor.java

```diff
@@ -323,166 +328,165 @@ private void flagFiles(Set<String> fileSet, String pluginName) {
+     * @param pluginName   Name of volatility module run
      * @param PluginOutput File that contains the output to parse
      */
     private void scanOutputFile(String pluginName, File PluginOutput) {
-           
+
         if (pluginName.matches("dlllist")) {
-           Set<String> fileSet = parseDllList(PluginOutput);
-           flagFiles(fileSet, pluginName);
+            Set<String> fileSet = parseDllList(PluginOutput);
+            flagFiles(fileSet, pluginName);
         } else if (pluginName.matches("handles")) {
-           Set<String> fileSet = parseHandles(PluginOutput);
-           flagFiles(fileSet, pluginName);
-        } else if (pluginName.matches("cmdline")) { 
-           Set<String> fileSet = parseCmdline(PluginOutput);
-           flagFiles(fileSet, pluginName);
-        } else if (pluginName.matches("psxview")){
-           Set<String> fileSet = parsePsxview(PluginOutput);
-           flagFiles(fileSet, pluginName);
+            Set<String> fileSet = parseHandles(PluginOutput);
+            flagFiles(fileSet, pluginName);
+        } else if (pluginName.matches("cmdline")) {
+            Set<String> fileSet = parseCmdline(PluginOutput);
+            flagFiles(fileSet, pluginName);
+        } else if (pluginName.matches("psxview")) {
+            Set<String> fileSet = parsePsxview(PluginOutput);
+            flagFiles(fileSet, pluginName);
         } else if (pluginName.matches("pslist")) {
-           Set<String> fileSet = parsePslist(PluginOutput);
-           flagFiles(fileSet, pluginName);
-        } else if (pluginName.matches("psscan")) { 
+            Set<String> fileSet = parsePslist(PluginOutput);
+            flagFiles(fileSet, pluginName);
+        } else if (pluginName.matches("psscan")) {
             Set<String> fileSet = parsePsscan(PluginOutput);
             flagFiles(fileSet, pluginName);
         } else if (pluginName.matches("pstree")) {
-           Set<String> fileSet = parsePstree(PluginOutput);
-           flagFiles(fileSet, pluginName);
+            Set<String> fileSet = parsePstree(PluginOutput);
+            flagFiles(fileSet, pluginName);
         } else if (pluginName.matches("svcscan")) {
-           Set<String> fileSet = parseSvcscan(PluginOutput);
-           flagFiles(fileSet, pluginName);
+            Set<String> fileSet = parseSvcscan(PluginOutput);
+            flagFiles(fileSet, pluginName);
         } else if (pluginName.matches("filescan")) {
             // BC: Commented out.  Too many hits to flag
-           //Set<String> fileSet = ParseFilescan(PluginOutput);
-           //lookupFiles(fileSet, pluginName);  
-        } else if (pluginName.matches("shimcache")) { 
-           Set<String> fileSet = parseShimcache(PluginOutput);
-           flagFiles(fileSet, pluginName);
-        }        
-    } 
-
-    /** 
-     * Normalize the path we parse out of the output before
-     * we look it up in the case DB
-     * 
+            //Set<String> fileSet = ParseFilescan(PluginOutput);
```

Comment:
Either remove this part entirely (instead of commenting it out) or restore the ParseFilescan method (it's pointless to preserve the reference if the method is gone, right?)

Reply:
Will do.